src/bhavana_assignment_15/AccountNestedSwitch.java

```diff
@@ -0,0 +1,167 @@
+package bhavana_assignment_15;
+/*A person has total 4 accounts in ICICI, HDFC, BOB, SBI one in each.
+          a) Person can debit or credit amount from respective bank.
+              Hint : Switch case is expected here. [Please watch today's recording again, in case of confusion.]
+
+          b) A Method to show total balance [ICICI balance + HDFC balance + BOB balance + SBI Balance]
+
+          c) A Method to show total credit, debit operations 
+                 output : Total credit operation - 7 times
+                              Total debit operation - 5 times
+
+         d) A Method to show individual bank credit operations.
+                output : ICICI bank Credit Operation - 2
+                         HDFC bank Credit Operation - 2
+		                 SBI bank Credit Operation - 1
+                         BOB bank Credit Operation - 4
+
+         e) A Method to show individual bank debit operations.
+                output : ICICI bank Debit Operation - 2
+                         HDFC bank Debit Operation - 2
+		                 SBI */
+
+public class AccountNestedSwitch {
+	
+	double iciciBal,hdfcBal,bobBal,sbiBal,totalBalance;
+	int totalDebitCount,totalCreditCount;
+	int iciciDebitCount,hdfcDebitCount,bobDebitCount,sbiDebitCount;
+	int iciciCreditCount,hdfcCreditCount,bobCreditCount,sbiCreditCount;
+	String custName;
+	
+	void setCustomerDetails(String name) {
+		custName=name;
+		System.out.println(custName+"'s details are:");
+	}
+	
+	void setInitialBalance(double icici,double hdfc,double bob,double sbi) {
+		iciciBal=icici;
+		hdfcBal=hdfc;
+		bobBal=bob;
+		sbiBal=sbi;
+	}
+	
+	
+	void debitCreditOperation(String bank, String op,double amount) {
+		switch(op) {
+			case"debit":
+				switch(bank) {
+					case"icici":
+						if(iciciBal>=amount) {
```

Comment:
instead of writing same code block for multiple times, you can write debit() method which accept the two arguments and you can simply call that method.

Code snippet

int debit(int totalAmt, int debitAmt) {
		if (totalAmt < debitAmt) {
			return 0;
		} else
			return totalAmt - debitAmt;
}

totalAmt will your iciciAmt (instance variable)